client/verta/verta/tracking/entities/_deployable_entity.py

```diff
@@ -448,7 +494,11 @@ def fetch_artifacts(self, keys):
             #     identical, so multiple writes would be idempotent.
             path = self._get_cached_file(filename)
             if path is None:
-                contents = self._get_artifact(key)
+                try:
+                    contents = self._get_artifact(key)
+                except ValueError as error:
+                    logger.exception(error)
```

Comment:
A few thoughts on this:
1. `logger.exception()` already [automatically captures exception info](https://docs.python.org/3/library/logging.html#logging.Logger.exception) (specifically the stack trace), so I think attaching the exception as the message may be redundant
2. This is a bit shakier, but to me this isn't exactly an exception when fetching the artifact per se (it's more of a prior exception on _upload_ that left it in a bad state for subsequent operations) so I'd consider a warning more appropriate than an exception (and `warning()` doesn't do the automatic thing from (1), so the exception needs to be given).
3. I'd request using `as e` which is what [most of] the rest of our codebase is using today

```suggestion
                except ValueError as e:
                    logger.warning(exc_info=e)
```

Reply:
@daniel-verta Actually, `True` can be passed to `exc_info` which will fetch the stack trace from the context, so we wouldn't have to bother with the `as e` at all!

```suggestion
                except ValueError:
                    logger.warning(exc_info=True)
```